src/views/SignUp.vue

```diff
@@ -0,0 +1,28 @@
+<template lang="pug">
+form-component(:fieldsArray='fieldsArray' :handleAction='handleCreateUser' :formData='userNew' )
```

Comment:
Use kebab-case notation for all props and attributes in the template
`:name-prop='nameProp'`